cfy-go/info.go

```diff
@@ -157,39 +158,70 @@ func groupInstancesChecks(cl *cloudify.Client, params map[string]string) int {
 }
 
 func runChecks(params map[string]string) int {
+	res := runChecksStatus() != 1 && runChecksNode(params) != 1 && runChecksLoad(params) != 1 && runChecksScaleGroup(params) != 1
+	if res {
+		return 0
+	}
+	return 1
+}
+
+func runChecksNode(params map[string]string) int {
 	cl := getClient()
 	var res int
-
-	fmt.Println("* Check manager services status.")
-	fmt.Println("  Recheck by 'cfy-go status state'")
-	res = servicesPrint(cl.GetStatus())
-	if res != 0 {
-		return res
+	var nodeType = os.Getenv("CFY_K8S_NODE_TYPE")
+	if nodeType == "" {
+		nodeType = cloudify.KubernetesNode
 	}
 
 	fmt.Println("* Check properties in kubernetes instances.")
 	fmt.Println("  Recheck by 'cfy-go node-instances started'")
-	res = instancesChecks(cl, params, cloudify.KubernetesNode,
-		[]string{"ip", "public_ip"})
+	res = instancesChecks(cl, params, nodeType, []string{"ip", "public_ip"})
 	if res != 0 {
 		return res
 	}
+	return 0
+}
+
+func runChecksLoad(params map[string]string) int {
+	cl := getClient()
+	var res int
+	var loadType = os.Getenv("CFY_K8S_LOAD_TYPE")
+	if loadType == "" {
+		loadType = cloudify.KubernetesLoadBalancer
+	}
 
 	fmt.Println("* Check properties in kubernetes loadbalancers.")
 	fmt.Println("  Recheck by 'cfy-go node-instances loadbalancer'")
-	res = instancesChecks(cl, params, cloudify.KubernetesLoadBalancer,
-		[]string{"ip", "public_ip", "proxy_cluster", "proxy_namespace", "proxy_name"})
+	res = instancesChecks(cl, params, loadType, []string{"ip", "public_ip", "proxy_cluster", "proxy_namespace", "proxy_name"})
```

Comment:
Use shorter lines, diff in github hide line endings.

Reply:
Done